sdks/java/extensions/sql/build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
   shadow library.java.joda_time
   shadow project(path: ":beam-runners-direct-java", configuration: "shadow")
   provided project(path: ":beam-sdks-java-io-kafka", configuration: "shadow")
+  provided project(path: ":beam-sdks-java-io-google-cloud-platform", configuration: "shadow")
```

Comment:
<!--new_thread; commit:a96a74cc97c889d1b604cf399fe0890f68f41c8b; resolved:1-->
Commented on the pom, but since it is deprecated, consider that comment to apply to this line.